README.md

```diff
@@ -88,6 +88,16 @@ With that in mind, the steps for adding a new API are:
 4. Wait for CI to pass, fixing errors.
 5. Wait for a merge!
 
+### Test before you commit
+
+We have two automated tests running on [Travis](https://travis-ci.org/rust-lang/libc):
+
+1. `libc-test`
+  - `cd libc-test && cargo clean && cargo run`
```

Comment:
OK, confirmed, but

1. Please update the lock file on your side. In a sense you chose not to release the fix on `rerun-if-changed` by distributing the lock file.
2. I found the semantic version of a Git dependency confusing. I filed rust-lang/cargo#2586